modules/junit-jupiter/src/test/java/org/testcontainers/junit/jupiter/MixedLifecycleTests.java

```diff
@@ -0,0 +1,37 @@
+package org.testcontainers.junit.jupiter;
+
+import org.apache.http.HttpResponse;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.DockerComposeContainer;
+import org.testcontainers.containers.MySQLContainer;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+
+// testClass {
+@Testcontainers
+class MixedLifecycleTests {
+
+    // will be shared between test methods
+    @Container
+    private static final MySQLContainer MY_SQL_CONTAINER = new MySQLContainer();
+
+    // will be started before and stopped after each test method
+    @Container
+    private PostgreSQLContainer postgresqlContainer = new PostgreSQLContainer()
+        .withDatabaseName("foo")
+        .withUsername("foo")
+        .withPassword("secret");
+    @Test
```

Comment:
Let's just squeeze in a newline to create some visual separation

```suggestion

    @Test
```